### PR TITLE
Add missing NEWS for 1.2.0

### PR DESCRIPTION
**Pull request by ajreynol**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -9,14 +9,40 @@ cvc5 1.2.0
...
+- Exposed creation and maintenance of **Skolem functions** to the API. Skolem
+  functions are symbols that are internally introduced by cvc5 during solving.
+  They are formalized via an identifier (see
+  https://cvc5.github.io/docs-ci/docs-main/skolem-ids.html for details) as well
+  as a (possibly empty) set of indices.
+  + The **API** methods `Term::isSkolem()`, `Term::getSkolemId()`, and
+    `Term::getSkolemIndices()`may now be used to identify terms corresponding
+    to skolems.
+
+- Add support for the kind `BITVECTOR_NEGO` corresponding to bitvector
```
Comment (aniemetz) [inline NEWS.md:21]: This was new in 1.1.2, maybe move it there (and also update the release notes for 1.1.2 on GH).

Comment (ajreynol) [reply NEWS.md:21]: Sounds good.

File: NEWS.md [hunk 2]
```diff
@@ -9,14 +9,37 @@ cvc5 1.2.0
...
+
+- Proof rules corresponding to rewrite rules are now exported in the API via
+  the enum `ProofRewriteRule`.
+
+- A new `Plugin` class is available in our API. This class allows a user
+  to be notified when SAT clauses or theory lemmas are learned internally
+  by cvc5. The plugin class also contains a callback for the user to introduce
+  new learned clauses into the state of cvc5 during solving.
+
+- Adds a new strategy `--mbqi-fast-sygus` (disabled by default) for **quantifier
```
Comment (aniemetz) [inline NEWS.md:33]:
```suggestion
- Added a new strategy `--mbqi-fast-sygus` (disabled by default) for **quantifier
```

File: NEWS.md [hunk 3]
```diff
@@ -42,6 +65,10 @@ cvc5 1.2.0
   * The rules of this format have been formalized in Eunoia and are available
     in the cvc5 repository under the directory `./proofs/eo/cpc/`.
 
+- The semantics of `SQRT` was changed to assume the result is positive.
+
+- Fixes a bug involving how sequence terms are printed in model values.
```
Comment (aniemetz) [inline NEWS.md:70]:
```suggestion
- Fixed a bug involving how sequence terms are printed in model values.
```

**Review by aniemetz — APPROVED** [review]

**Pull request merged**